cypress/e2e/specialFeatures/atiAnalytics/index.cy.js

```diff
@@ -474,8 +474,6 @@ const liteTestSuites = canonicalTestSuites
     const liteSiteTests = testSuite.tests.filter(
       test =>
         test.name !== assertMostReadComponentClick.name &&
```

Comment:
I think we can also remove this time, as the Most Read Component Click will now be included